python/lsst/ts/fbs/utils/auxtel/basis_functions.py

```diff
@@ -203,7 +203,4 @@ def get_basis_functions_spectroscopic_survey(
         basis_functions.AvoidDirectWind(
             wind_speed_maximum=wind_speed_maximum, nside=nside
         ),
```

Comment:
Can you update the unit test?

It is failing because it expects a certain number of basis functions:

```
tests/auxtel/test_basis_functions.py::test_get_basis_functions_spectroscopic_survey
```